src/app/EmployeesHistory/Components/employeeSummary.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import ItemsList from './itemsList';
+import EmployeeHistoryProvider from '../employeeHistoryProvider';
```

Comment:
Files containing utility functions, constans would be better of in an util/constants folder.
Also, these files sould export constant functions rather than a default export.